aws/cloudformation/iam/security_audit.yml

```diff
@@ -0,0 +1,35 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
I have always wonder why we use this version, is this the latest one? Does anyone know if we can move to a more recent date?